src/apiRedux/dishes.js

```diff
@@ -1,51 +1,42 @@
 import { config } from '../config'
 
-export const getDishes = async () => {
-  const res = await fetch(`${config.url}/dishes`)
-  return await res.json()
-}
-
-export const getDishesCategories = async () => {
-  const res = await fetch(`${config.url}/dishesCategories`)
-  return await res.json()
-}
-
-export const getBasket = async () => {
-  const res = await fetch(`${config.url}/basketDishes`)
-  return await res.json()
-}
-
-export const postBasketDishes = async ({ data }) => {
-  const response = await fetch(`${config.url}/basketDishes`, {
-    method: 'POST',
-    headers: {
-      'Content-type': 'application/json',
-    },
-    body: JSON.stringify(data),
-  })
-
-  return await response.json()
-}
-
-export const putBasketDishes = async ({ data, id }) => {
-  const response = await fetch(`${config.url}/basketDishes/${id}`, {
-    method: 'PUT',
-    headers: {
-      'Content-type': 'application/json',
-    },
-    body: JSON.stringify(data),
-  })
-
-  return await response.json()
-}
-
-export const deleteBasketDishes = async ({ id }) => {
-  const response = await fetch(`${config.url}/basketDishes/${id}`, {
-    method: 'DELETE',
-    headers: {
-      'Content-type': 'application/json',
-    },
-  })
-
-  return await response.json()
+import { getPOSTparams, getPUTparams, getDELETEparams } from 'assets'
```

Comment:
takie funkcje lepiej umieścić w przypadku tej apki w src/apuRedux/helpers.js - uzywasz ich tylko w api, nie będą potrzebne poza nim.

Reply:
nie rozumiem o co Ci chodzi.
Do omówienia na hengouts.

Reply:
chodzi mi o umiejscowienie tego pliku z tymi funkcjami - samo assets w roocie jest troche bez sensu, bo nie jest to powszechnie uzywany plik. pasuje do folderu z api, jako miejscowy helper

Reply:
True story boro.
Jobs done.
https://www.youtube.com/watch?v=5r06heQ5HsI&ab_channel=PinchySkree